openlibrary/catalog/marc/parse.py

```diff
@@ -279,14 +279,39 @@ def read_edition_name(rec: MarcBase) -> str:
     'end': 'eng',
     'enk': 'eng',
     'ent': 'eng',
-    'cro': 'chu',
     'jap': 'jpn',
     'fra': 'fre',
-    'gwr': 'ger',
-    'sze': 'slo',
-    'fr ': 'fre',
     'fle': 'dut',  # Flemish -> Dutch
+    # 2 character to 3 character codes
+    'fr ': 'fre',
     'it ': 'ita',
+    # LOC MARC Deprecated code updates
+    'cam': 'khm',  # Khmer
+    'esp': 'epo',  # Esperanto
```

Comment:
I only added the codes which had a clear one-to-one correct mapping -- your list was super helpful showing the corrected mappings. Many deprecated codes don't have a single obvious mapping, which prevents such mapping, and there seems to be a range of reasons why a code is deprecated (some seem technical dialect vs language factors like `lan` -> `oci` -- that makes me think some cataloged items might lose information if the item is really in the Languedocien dialect and were cataloged correctly, but now they'd be listed under a family (Occitan), with a time period, which may or may not relate to the item. It could go the other way though. I don't know enough of the details, but that struck me as potentially quite a difference.

  I think your advice on what is needed to correct the ~217 `esk` codes in https://github.com/internetarchive/openlibrary/issues/8733#issuecomment-1901168076 is still good, and I'm not sure it can be automated (without the risk of mis-assigning codes based on naive assumptions).

Reply:
It might be worth tossing in a comment about `esk` (and any other unmappable codes) so that people reviewing the list don't think that they were forgotten.